tracer/test/Datadog.Trace.TestHelpers/TestHelper.cs

```diff
@@ -661,6 +675,86 @@ protected void EnableTelemetry(bool enabled = true, int? standaloneAgentPort = n
             Assert.Equal(expectedServiceVersion, span.Tags.GetValueOrDefault(Tags.Version));
         }
 
+        protected void SetupRcmConfiguration(string path = null)
```

Comment:
I think the name of method is misleading.
Method is setup environment variables for RCM, so probably use `SetupRcmEnvironment` or simply `SetupRcm`?

Reply:
SetupRcm sounds good. I buy it :)

Reply:
Done in c5cf6dc713be9d83186bc89361d8c9524215252e

Reply:
Should I switch from engineering to sales? 😆